spec/javascripts/Common/components/TableToolbar.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { mount } from 'enzyme'
+
+import { TableToolbar } from 'Common/components/TableToolbar'
+
+import type { Props } from 'Common/components/TableToolbar'
+
+const defaultProps = {
+  totalEntries: 50,
+  leftActions: undefined,
+  rightActions: undefined,
```

Comment:
```suggestion
  actions: undefined,
  overflow: undefined,
```

I realize that this is TBD, but just to get rid of confusion on the props names.